File: square/src/main/java/com/raxdenstudios/square/interceptor/network/NetworkInterceptorCallback.java
package com.raxdenstudios.square.interceptor.network;

import com.raxdenstudios.square.interceptor.InterceptorCallback;

/**
 * Created by agomez on 08/05/2015.
 */
public interface NetworkInterceptorCallback extends InterceptorCallback {

    void onWifiAvailable(boolean available);

    void onNetworkAvailable(boolean available);

}

File: square-commons/src/main/java/com/raxdenstudios/square/interceptor/commons/network/NetworkInterceptorCallback.kt
package com.raxdenstudios.square.interceptor.commons.network

import com.raxdenstudios.square.interceptor.InterceptorCallback

/**
 * Created by agomez on 08/05/2015.
 */
interface NetworkInterceptorCallback : InterceptorCallback {

    fun onWifiAvailable(available: Boolean)

    fun onNetworkAvailable(available: Boolean)

}
